features/storage/FEATURE_STORAGE/cfstore/source/cfstore_config.h

```diff
@@ -49,8 +49,11 @@
  *   Disable use of storage support (if present)
  */
 #if defined DEVICE_STORAGE && CFSTORE_STORAGE_DISABLE==0
+#ifdef TARGET_K64F
```

Comment:
As @dannybenor mentioned above, cfstore is deprecated.

Reply:
Thanks. I fixed it on e578320f5aea90759f813e9a6f1eea2d2f884936.